frontend/packages/core/src/stories/button.stories.tsx

```diff
@@ -14,13 +14,25 @@ export default {
 
 const Template = (props: ButtonProps) => <Button {...props} />;
 
-export const Default = Template.bind({});
-Default.args = {
-  text: "continue",
+export const Primary = Template.bind({});
+Primary.args = {
+  text: "Continue",
 };
 
 export const Destructive = Template.bind({});
 Destructive.args = {
-  text: "delete",
-  destructive: true,
```

Comment:
should we call this one "Caution" too, like in button group?

Reply:
Reverted back to destructive